docker/Dockerfile_dev

```diff
@@ -1,7 +1,6 @@
-FROM python:3.8.15
+FROM python:3.8.15-slim
```

Comment:
why the slim python for the development container? it might be nice for the devcontainer workflow to have more than the minimal python docker image